### ISSUE TITLE
InvalidArgumentException: Expected a string. Got: ReflectionMethod

### ISSUE THREAD

**Issue by ahungry** [johnbillion/php-docs-standards#3]
I've tried this out on a few files that are in classes within my Symfony project.

It seems to parse the methods within the class, however instead of doing anything useful, each DataProvider map across those methods simply returns:

InvalidArgumentException: Expected a string. Got: ReflectionMethod

This is with PHP7

**Comment by johnbillion** [comment]
I've seen this too. It's due to PHPUnit bundling a newer version of `phpdocumentor/reflection-docblock`.

Can you try running your tests with `./vendor/bin/phpunit` (instead of just `phpunit`) and let me know if the issue goes away?

**Comment by ahungry** [comment]
It did fix the issue, thanks!

Do you plan to update the code in php-docs-standards to work with the newer version eventually, or will it be locked to the older version?

**Comment by johnbillion** [comment]
Yep. I'm planning to update the library to allow it to use static analysis instead too. See #1.

**Comment by johnbillion** [comment]
I'll get the docs updated to mention the PHPUnit issue.
